ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_additional_paths.schema.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+type: dict
+$defs:
+  additional_paths:
+  type: dict
+  keys:
+    receive:
+      type: bool
+    send:
+      type: str
+      description: |-
+        Select an option to send multiple paths for same prefix through bgp updates.
+        any: Send any eligible path.
+        backup: Best path and installed backup path.
+        ecmp: All paths in best path ECMP group.
+        limit: Limit to n eligible paths
+      valid_values: [ 'any', 'backup', 'ecmp', 'limit' ]
+    ecmp_limit:
+      type: int
+      description: Amount of ECMP paths to send. If this key-value pair is not set, it will send all paths in best ECMP group.
+      convert_types:
+        - str
+      min: 2
+      max: 64
+    limit:
+      type: int
+      description: Number of paths to send through bgp updates(The value of 'send' key must be 'limit').
```

Comment:
We may have to rename the ecmp_limit -> send_ecmp_limit and limit -> send_limit. Just in case there will alter come some receive limit options.

Reply:
Fixed.